src/formatDate.js

```diff
@@ -50,7 +50,42 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [, , , signFrom] = fromFormat;
```

Comment:
```suggestion
  const signFrom = fromFormat[3];
```
not critical, but in this case, it looks better